lms/djangoapps/grades/tasks.py

```diff
@@ -24,9 +26,27 @@
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
 def recalculate_subsection_grade(
+        # pylint: disable=unused-argument
         user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
 ):
     """
+    Shim to allow us to modify this task's signature without blowing up production on deployment.
+    """
+    _recalculate_subsection_grade.apply(
+        kwargs=dict(
+            user_id=user_id,
+            course_id=course_id,
+            usage_id=usage_id,
+            only_if_higher=only_if_higher,
+            task_queued_time=kwargs.get('task_queued_time', 0),  # If no time, assume 1/1/1970
```

Comment:
This is a leftover remnant from when I was toying with using unix timestamps, one second...